builtin/commit-graph.c

```diff
@@ -156,6 +160,8 @@ static int graph_write(int argc, const char **argv)
 	int result = 0;
 	unsigned int flags = COMMIT_GRAPH_PROGRESS;
 
+	trace2_cmd_mode("write");
```

Comment:
@jeffhostetler @derrickstolee  write takes a bunch of options. Are there any particularly interesting ones that I should be more granular about?

Reply:
I think if we want to get picky about arguments, then it is better to start parsing the full argument data instead of relying on custom trace2 calls here. Splitting the subcommands is enough for us to get the "big" categories.

Plus: for a given version of VFS for Git, the rest of the arguments will be the same (`verify --shallow` always, `write --split --stdin-packs` always)

Reply:
Thanks! "Parsing the full argument data" as in - in the Kusto traces yes?

Reply:
I think it is sufficient to just let us easily distinguish between "write" and "verify" and etc to make queries easier.